### DISCUSSION TITLE
如何更改可执行文件的输出路径

### DISCUSSION THREAD

**Discussion by CuiZhenhang** [cpeditor/cpeditor-cn#22] [提问] [answered]
我使用的是 6.9.4  setup 版本（Windows 10，64 位），当前可执行文件的输出路径为类似 `%TEMP%\CP Editor-AdzdoK` 的路径。
我的杀毒软件有时会在我运行时询问“是否阻止可疑行为”，导致 CP Editor 内显示执行失败。
我的杀毒软件可以添加信任区，但由于可执行文件的输出路径是经常变化的，而且将 `%TEMP%` 文件夹加入信任区是危险的，导致我无法解决该问题。
如果可执行文件的输出路径为类似 `%TEMP%\CP Editor\CP Editor-AdzdoK` 的路径，那么该问题就可以解决了。
但我没能找到更改可执行文件的输出路径至这样的方法。

**Comment by ouuan** [comment, marked as answer]
可以修改可执行文件路径: <https://cpeditor.org/zh/docs/preferences/language/#c-可执行文件路径>

但目前确实无法修改为类似 `%TEMP%\CP Editor\CP Editor-AdzdoK` 的路径，只能要么 `%TEMP%\CP Editor-AdzdoK` 要么不在 `%TEMP%` 目录下（不会自动删除）。
